stylesheets/_component.daterange.scss

```diff
@@ -1,204 +1,298 @@
-.daterangepicker.dropdown-menu {
-  max-width: none;
-  z-index: 3000;
+// scss-lint:disable SelectorFormat
+daterangepicker.dropdown-menu {
+    max-width: none;
+    z-index: 3000;
```

Comment:
Can we replace this with a z-index variable? (I realise it was there before you started)